lib/pbench/agent/tool_meister.py

```diff
@@ -1051,7 +1043,7 @@ def _send_client_status(self, status: str) -> int:
                 ret_val = 0
         return ret_val
 
-    def init_tools(self, data: dict) -> int:
+    def init_tools(self, data: dict[str, str]) -> int:
```

Comment:
Again, technically `dict[str, str]` isn't right. Whether our tools will complain is a different matter, but it should be `Dict[str, str]`.

[https://peps.python.org/pep-0585/]

> This is why starting with Python 3.9, the following collections become generic using `__class_getitem__()` to parameterize contained types: [... , including `dict`]

(Also Python 3.7 with `__future__` defined; but we're still supporting Python 3.6, so we need to use the `typing` collection generics.)

Reply:
BTW, I'm confused by the log message for this latest commit, "\`Dict\` not \`dict\`", since I don't see any occurrences of `Dict` in the change...but, if we squash this branch, then that won't be a problem....

Reply:
FWIW, my local `.patch` file for that commit has those changes.